workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -133,6 +136,41 @@ def add_tendency(state: Any, tendency: State, dt: float) -> Tuple[State, State]:
     return updated, tendency_filled_frac  # type: ignore
 
 
+def _download_radiation_assets(
+    lookup_data_path: str = (
+        "gs://vcm-fv3gfs-serialized-regression-data/physics/lookupdata/lookup.tar.gz"  # noqa: E501
+    ),
+    forcing_data_path: str = "gs://vcm-fv3gfs-serialized-regression-data/physics/forcing/*",  # noqa: 501
+    lookup_local_dir: str = "./data/lookup",
+    forcing_local_dir: str = "./data/forcing",
+) -> None:
+    """Gets lookup tables and forcing needed for the radiation scheme. TODO: Make lookup
+    data part of writing a run directory; make scheme able to read existing forcing.
+    """
+    os.makedirs(lookup_local_dir)
+    copy_cmd = ["gsutil", "cp", lookup_data_path, lookup_local_dir]
```

Comment:
Can you avoid shelling out to system utilities in this script? `tarfile` is a python built in library, and you can use fsspec for the downloading.

Reply:
Sure